### PR TITLE
feat: add ? keyboard shortcut for showing known shortcuts

### PR DESCRIPTION
**Pull request by vapier**

### CODE REVIEW

**Comment by vapier** [comment]
not sure about the style in the HTML.  should i unpack them so each <tr> takes up multiple lines ?

File: webui-page/index.html [hunk 1]
```diff
@@ -55,6 +55,27 @@
...
+        <tr><td>p</td><td>playlist previous</td></tr>
+        <tr><td>[</td><td>decrease playback speed</td></tr>
+        <tr><td>]</td><td>increase playback speed</td></tr>
+        <tr><td>{</td><td>decrease playback speed more</td></tr>
+        <tr><td>}</td><td>increase playback speed more</td></tr>
+        <tr><td>Backspace</td><td>reset playback speed</td></tr>
+        <tr><td>Escape</td><td>hide current overlay</td></tr>
+        <tr><td>?</td><td>toggle keyboard shortcuts overlay</td></tr>
+      </table>
+    </div>
```
Comment (open-dynaMIX) [inline webui-page/index.html:77]: If we'd have the shortcut configuration globally available in `webui.js`, we would be able to generate the shortcut table. The we'd have a single location for configuring shortcuts.

**Comment by vapier** [comment]
i've overhauled things as suggested

**Comment by open-dynaMIX** [comment]
Awesome!

**Pull request merged**